pkg/networks/commands_test.go

```diff
@@ -26,15 +26,17 @@ func TestVDESock(t *testing.T) {
 	assert.NilError(t, err)
 
 	vdeSock := config.VDESock("foo")
-	assert.Equal(t, vdeSock, "/private/var/run/lima/foo.ctl")
+	varRunDir := filepath.Join("/", "private", "var", "run", "lima")
```

Comment:
This is obvious specific to macOS. We can move the test to `_darwin.go` without changing the code.

Reply:
Not sure if you meant the test, or the whole file. The code is not specific to darwin build though ?